centreon/lang/fr_FR.UTF-8/LC_MESSAGES/messages.po

```diff
@@ -18916,3 +18916,9 @@ msgstr "Standard"
 
 msgid "Condensed"
 msgstr "Condensé"
+
+msgid "The contact group [%s] is not in your access groups"
+msgstr "Le groupe de contact [%s] ne fait pas partie de vos groupes d'accès"
+
+msgid "The user [%s] is not in your access groups"
+msgstr "L'utilisateur [%s] ne fait pas partie de vos groupes d'accès"
```

Comment:
I'm not a fan of the wording, we usualy just say that the user/usergroup doesn't exist or was not found without specifying that its due to the user rights.